internal/perf/perf.go

```diff
@@ -909,9 +1000,10 @@ func (pr *perfRunner) runLoop(tc TestCase) error {
 	}
 }
 
+// TODO at the option of batching here!
 func (pr *perfRunner) createMsgConfirmSub(nodeURL, name, tag string) (subID string, subName string, err error) {
 	var sub core.Subscription
-	readAhead := uint16(len(pr.wsReceivers))
+	readAhead := uint16(len(pr.wsReceivers)) // TODO this makes no sense as it's arbitrary from what I can see
```

Comment:
As per my understanding, the number of WS receivers equals to the number of submitters. Therefore, the original intent is to control the read ahead to not get more event than submitted transactions. This might be a naive design but it was to some level configurable.